incubator/dokuwiki/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.image }}"
+        imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+        env:
+        - name: DOKUWIKI_USERNAME
+          value: {{ default "" .Values.dokuwikiUsername | quote }}
+        - name: DOKUWIKI_FULLNAME
```

Comment:
I think in bitnami/dokuwiki image this variable should have been named `DOKUWIKI_FULL_NAME` as the argument to the nami module is `--fullName`. Having a consistent naming convention would help in standardizing the entrypoints in the future /cc @prydonius